src/main/java/com/smartbear/readyapi/client/teststeps/TestSteps.java

```diff
@@ -7,34 +7,38 @@
 import com.smartbear.readyapi.client.teststeps.groovyscript.GroovyScriptTestStepBuilder;
 import com.smartbear.readyapi.client.teststeps.jdbcrequest.JdbcConnection;
 import com.smartbear.readyapi.client.teststeps.propertytransfer.PropertyTransferTestStepBuilder;
-import com.smartbear.readyapi.client.teststeps.restrequest.RestRequestBuilder;
-import com.smartbear.readyapi.client.teststeps.restrequest.RestRequestBuilderWithBody;
-import com.smartbear.readyapi.client.teststeps.restrequest.RestRequestTestStepBuilder;
+import com.smartbear.readyapi.client.teststeps.restrequest.RestRequestStepBuilder;
+import com.smartbear.readyapi.client.teststeps.restrequest.RestRequestStepWithBodyBuilder;
+import com.smartbear.readyapi.client.teststeps.soaprequest.SoapRequestStepBuilder;
 
 public class TestSteps {
 
     public enum HttpMethod {
         GET, POST, PUT, DELETE, HEAD, OPTIONS, TRACE, PATCH
     }
 
-    public static RestRequestTestStepBuilder restRequest() {
-        return new RestRequestTestStepBuilder();
+    public static SoapRequestStepBuilder soapRequest(String wsdl) {
+        return new SoapRequestStepBuilder().withWsdl(wsdl);
     }
 
-    public static RestRequestBuilder getRequest(String uri) {
-        return new RestRequestTestStepBuilder().get(uri);
+    public static RestRequestStepBuilder restRequest() {
+        return new RestRequestStepBuilder(null, TestSteps.HttpMethod.GET);
```

Comment:
I think you can still create a RestRequestStepBuilder manually and override method/body/etc...